unity/Assets/Scripts/Unity/UnityMap.cs

```diff
@@ -33,27 +35,32 @@ private void Awake()
 
         private void Start()
         {
-            unityStack = GameObject.FindGameObjectWithTag("Stack").GetComponent<UnityStack>();
-            unityPoints = GameObject.FindGameObjectWithTag("Points").GetComponent<UnityPoints>();
+            unityStack = GameObjectFinder.Stack.GetComponent<UnityStack>();
```

Comment:
Why was this changed again, this was recently changed to `FindGameObjectWithTag`

Reply:
I ended up using the previous, regular FindGameObjectWithTag statements multiple times which is error prone and quickly becomes a hassle to change. With the new central way of defining it once in the GameObjectFinder class, we can change whatever we like once and have it be consistent across any usage case.